src/Combinatorics/EnumerativeCombinatorics/combinations.jl

```diff
@@ -18,7 +18,7 @@ julia> collect(C)
  [2, 3, 4]
 ```
 """
-combinations(n::T, k::T) where T<:IntegerUnion = Combinations(Base.oneto(n), n, k)
+combinations(n::T, k::T; inplace::Bool = false) where T<:IntegerUnion = Combinations(Base.oneto(n), n, k, inplace)
```

Comment:
Should we document this `inplace` keyword? I notice it is not documented for `weak_compositions` right now. But I'd argue it should be documented for all of these, as it is extremely useful for certain applications. The same explanation text could be shared by all affected functions.